src/core.js

```diff
@@ -1,6 +1,16 @@
 'use strict'
 const encodings = new Set(['json', 'buffer', 'string'])
 
+const { stateEncode } = require('./common')
```

Comment:
the common lib contains a reference to `Buffer` so we can’t load it into core.js because it’ll inject the Buffer polyfill into the browser.

I don’t actually see any functions in common.js that are used in both this file and node.js, so you should probably just move each of these into core.js and nodejs.js and then `rm` common.js